data/events/scripts/player.lua

```diff
@@ -86,6 +86,25 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	-- Store Inbox
```

Comment:
backwards compatibility is not the point here, show me any other server system that are breakable by modifing in a script, there is none because it's not design project have been doing, thats the main point.